cleanrl/sac_continuous_action.py

```diff
@@ -104,26 +127,56 @@
     env = Monitor(env, f'videos/{experiment_name}')
 
 # ALGO LOGIC: initialize agent here:
+# Weight init scheme
```

Comment:
The code seems a little messy for the initialization method. Consider taking a look at how this is done in PPO https://github.com/vwxyzjn/cleanrl/blob/65d0e4a1c3be60fd590b30108869b764990a2cb3/cleanrl/ppo_continuous_action.py#L221.

Reply:
Not just a little bit messy haha. Tidied up a little bit.
Removing the `if isinstance(...)` line would require to change to model definition into something like `self.net = nn.Sequential(...)` in both actor and critic ...